src/views/StatsView.vue

```diff
@@ -9,7 +9,7 @@
   <template v-else>
     <div id="stats-wrapper">
       <!-- Upper Service -->
-      <section id="upper-surface" class="pa-8 mb-6">
+      <section id="upper-surface" class="px-8 pt-8 mb-6">
```

Comment:
This removes the bottom padding on the top section (below the dates for the selected season) to change this:
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/49c6bd6b-9b53-417a-9b2e-7958fd65fd60)

Into this
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/ba1b1a13-ebe4-4a51-98f4-1f76945c7a3a)